pkg/proxy/proxy.go

```diff
@@ -102,6 +102,8 @@ type Config struct {
 	Log *log.Logger
 	// Force use of unencrypted ws:// protocol instead of wss://
 	NoWSS bool
+	// Override default timeout
+	Timeout int64
```

Comment:
could you please explain why we need to pass this in proxy.config?

Reply:
ah, i was thinking it would be more consistent with how the other hidden flags' values are passed (like noWSS and apiBaseUrl are passed this way into proxy as well).